### DISCUSSION TITLE
emmet table 5 5

### DISCUSSION THREAD

**Discussion by Southbaywebs** [phcode-dev/phoenix#1897] [Ideas]
{
  "Table Generator": {
    "prefix": "table",
    "body": [
      "<table>",
      "  <tr>",
      "    $1", // Generates table headers if you want.
      "  </tr>",
      "  ${2:repeatRow(${3}, ${4})}", // Loops for rows and columns.
      "</table>"
    ],
    "description": "Generate a table with a custom number of rows and columns"
  }
}

I need something like this for generating tables... Can we do user snippets? Also a properties panel would be nice like dreamweaver has and drag and dropping a .css stylesheet onto the code inspector to make it automatically fill out the <link data..

**Comment by abose** [comment]
There is an emmet extension in the extension store. Would that work for this?

**Comment by Southbaywebs** [comment]
Possibly yes, just having a hard time running a command to open the
snippets.JSON file. I did not see there was a command pallet like
cmd+shift+p like visual code has, that auto completes? I want to make my
own table 5 5 emmet, where it creates a table with 5 rows and 5 columns. I
tried googling where this file is, or how to customize my own snippets, but
the information was hard to find! Also, a visual insert button like
dreamweaver has would be useful for those who need to use tables.

I wrote about it here in the shortcomings of DW: https://sites99.com/

Thanks,
Kyle

On Mon, Oct 21, 2024 at 7:55 PM Arun Bose ***@***.***> wrote:

> There is an emmet extension in the extension store. Would that work for
> this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/phcode-dev/discussions/1897#discussioncomment-11011611>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJNU3TVBHTLGM6OJSLJAQNDZ4W5AXAVCNFSM6AAAAABQLJ2YDKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBRGE3DCMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Kyle Scott
Owner & CEO
360-870-1081
https://southbaywebs.com <http://southbaywebs.com>
***@***.***

**Reply by abose** [reply]
Got it. We are adding git and terminal support this release, so most work was on that.

 After that, there are plans for live preview edit functionalities, but UX in a way that is similar to figma for HTML than with Dreamweaver.

**Reply by abose** [reply]
Thanks for the link. Gives some insights into workflows. Do drop more feature requests or opinions as discussions. Usually we address most within a few releases.

**Reply by charlypa** [reply]
@Southbaywebs you may try an extension named `codebycode` even though it may not fit all your use cases

**Comment by Southbaywebs** [comment]
Okay thank you. Also phoenix out of the box does not auto fill in quotes ""
as well as dreamweaver does. Just another suggestion. I am excited for the
project though. I am stuck using Dreamweaver 2020 instead of 2021, because
2021 seems to have developed bugs! I will try to think of some more
feedback when I have time. I just hate how long Visual Code takes to
customize, but that is why alot "PRO" developers prefer it, because it is
so highly customizable. With Dreamweaver on its last legs, I have been
having hard time finding a replacement that works out of the box. I will
stick with dream wearing (2020 - long term support) as it has long term
support.

Not many people code with tables like me, so I am the minority and I do not
recommend you spending development hours/time implementing this feature!
Most do divs. Also, most coders do not even use inspector panels to insert
links like I do. I again, am the minority and would love to see a program
like Macromedia Fireworks that exports pure divs/js. I know this is just a
code editor, but we can only dream right. All I have right now is Rive.App
and AE and they both only support canvas, gif/mp4, etc, though rive has
plans to make non canvas workflows work, I think? Though if Dreamweaver
does die, I am in the market for a non industry replacement code editor
that works out the box and does not need customizing like Visual Code,
Webstorm etc. Mostly because I am a hobby coder and I do not want to spend
so much time fiddling with visual code and all the broken extensions!

On Tue, Oct 22, 2024 at 6:52 AM charly p abraham ***@***.***>
wrote:

> @Southbaywebs <https://github.com/Southbaywebs> you may try an extension
> named codebycode even though it may not fit all your use cases
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/phcode-dev/discussions/1897#discussioncomment-11017743>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJNU3TQYK7W67XBGJA5YKYDZ4ZKBDAVCNFSM6AAAAABQLJ2YDKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBRG43TIMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Kyle Scott
Owner & CEO
360-870-1081
https://southbaywebs.com <http://southbaywebs.com>
***@***.***

**Reply by abose** [reply]
Our thoughts exactly. We wanted to build an editor that works out of the box for most users working with web. That is the philosophy behind Phoenix Code, to make things simpler and not require extensive configs and plugins to do the basic things. Thanks for the notes, it helps!
